Source/CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ set(libdevilutionx_SRCS
   towners.cpp
   track.cpp
 
+  common/validation.cpp

```

Comment:
```suggestion
  items/validation.cpp
```

Reply:
Sounds good. Perhaps we could split up `items.cpp` into multiple files as well and place them here. It's quite massive.

Reply:
> Sounds good. Perhaps at another time we could split up `items.cpp` into multiple files as well and place them here. It's quite massive.